.changesets/feat_bnjjj_feat_otel_name_attribute.md

```diff
@@ -0,0 +1,16 @@
+### Override tracing span names using custom span selectors ([Issue #5261](https://github.com/apollographql/router/issues/5261))
+
+It gives you the ability to override the span name by using custom telemetry with any [selectors](https://www.apollographql.com/docs/router/configuration/telemetry/instrumentation/selectors/) you want just by setting the `otel.name` attribute.
+
+Example:
+
+```yaml
+telemetry:
+  instrumentation:
+    spans:
+      router:
+        otel.name:
+           static: router # Override the span name to router 
```

Comment:
What is the original span name? (I thought it was `router` and am therefore confused.)

Reply:
@bnjjj Can you answer this outside of this PR?  I'll resolve this so we can merge the release.

Reply:
It is router by default but if you have a GraphQL query with an operation name it's renamed with the operation name. This example is an example to show how you can enforce it to always keep router as span name